src/components/sync-configurator/BaseSyncConfigurator.jsx

```diff
@@ -124,7 +125,7 @@ class BaseSyncConfigurator extends React.Component {
             filters,
             importResponse,
         } = this.state;
-
+        const disabled = metadata.organisationUnit && _.isEmpty(metadata.organisationUnit);
```

Comment:
BaseSyncConfigurator is a generic component, we should try not to hardcode model names. Proposal for a more generic approach: 

```const disabled = _(metadata).values().some(_.isEmpty)```

I have some doubts about the type of `metadata`, we are using an object with has always has a single key. You usually would describe it simply with a pair (array) or an object. Example:

We have: ```const metadata = {organisationUnits: []}```
Option 1 (pair): ```const metadata = ["organisationUnits", []]```
Option 2 (object), usually the best: ```const metadata = {model: "organisationUnits", objects: []}```

Not that we have to change it, just a note for future code.

Reply:
The thing is that we are following how the import/export API works. The result of a metadata export is like that { metadataType: [] }. So we have it that way in code to reuse the response of DHIS2 without having to transform it.